snap-css-web/cypress.json

```diff
@@ -1,3 +1,4 @@
 {
-  "pluginsFile": "tests/e2e/plugins/index.js"
```

Comment:
better to not add cypress again
it has already been configured and the test:e2e script runs it properly and even handle running the server by itself, configured base url and stuff
is there a reason you had to add it again?